tests/wpt/meta/MANIFEST.json

```diff
@@ -39015,6 +39015,23 @@
      ]
     ]
    },
+   "content-security-policy": {
+    "generic": {
+     "image_document_ignores_csp.html": [
```

Comment:
nit: WPT tests typically use hyphens between words in the test name and it seems upstream CSP tests use this pattern. It might be a good idea to rename the test before landing.